catalystwan/endpoints/cluster_management.py

```diff
@@ -24,12 +26,30 @@ class VManageDetails(BaseModel):
     status: str
 
 
+class ConnectedDevice(BaseModel):
+    model_config = ConfigDict(populate_by_name=True)
+    uuid: UUID
+    device_id: str = Field(serialization_alias="deviceId", validation_alias="deviceId")
+
+
+class VManageSetup(BaseModel):
+    model_config = ConfigDict(populate_by_name=True)
+    vmanage_id: Optional[str] = Field(default=None, serialization_alias="vmanageID", validation_alias="vmanageID")
+    device_ip: str = Field(default=None, serialization_alias="deviceIP", validation_alias="deviceIP")
```

Comment:
I guess these needs to be `Optional[str]` if we allow `None` as default

Reply:
actually vars not marked as optional should not have default None in this case, my bad